src/App.css

```diff
@@ -21,3 +21,7 @@ body {
   padding: var(--spacing-2);
   font-family: var(--gotham);
 }
+
+html {
+  font-size: 16px;
```

Comment:
We might want to let Fast know that we're using `rem` values for sizes. So, they need to make sure it corresponds to `16px` on their end.